src/main/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraService.scala

```diff
@@ -30,19 +31,25 @@ abstract class AgoraService extends HttpService with RouteHelpers {
   def namespacePermissionsRoute =
     matchNamespacePermissionsRoute(path) { (namespace, username) =>
       parameterMap { (params) =>
-        val entity = AgoraEntity(Option(namespace))
+        val agoraEntity = AgoraEntity(Option(namespace))
 
+        // Accept batch POST // TODO: move to transactional support
+        entity(as[List[AccessControl]]) { (listOfAccessControl) =>
+          post { requestContext =>
+            completeBatchNamespacePermissionsPost(requestContext, agoraEntity, listOfAccessControl, username, permissionHandlerProps)
+          }
+        } ~
```

Comment:
Making sure I understand this - if it's a batch request we process it using the route above, and if it's a single request it falls through to the routes below?